### ISSUE TITLE
label mapping and bert/roberta output

### ISSUE THREAD

**Issue by nishchay47b** [doc-analysis/DocBank#14]
I tried bert and Roberta models avaialable in the model zoo. However, I am got only LABEL_8 for all the tokens. I tried with simpletransformers and hugging face transformer library.

```python
from simpletransformers import ner
import pytesseract
import json
with open("bert_large_500k_epoch_1/config.json") as f:
    config = json.load(f)

model_args = ner.NERArgs()
model_args.config = config
model_args.labels_list = ["LABEL_0", "LABEL_1", "LABEL_2", "LABEL_3", "LABEL_4", "LABEL_5", "LABEL_6",
                          "LABEL_7", "LABEL_8", "LABEL_9", "LABEL_10", "LABEL_11", "LABEL_12"]
model = ner.NERModel(
    'bert',
    'bert_large_500k_epoch_1',
    args=model_args,
    use_cuda=False
)
predictions, raw_outputs = model.predict([pytesseract.image_to_string('closing-disclosure-H25B-1.aec3e9325e5b.png')])
print(predictions)
```

```python
from transformers import RobertaTokenizer, RobertaForTokenClassification, RobertaConfig
import torch
import pandas as pd

tokenizer = RobertaTokenizer.from_pretrained('roberta-base')
model = RobertaForTokenClassification.from_pretrained("roberta_large_500k_epoch_1")
model.eval()

df = pd.read_csv("10.tar_1701.04170.gz_TPNL_afterglow_evo_8.txt", sep='\t', header=None)
text = " ".join(df[0].to_list())
sample_input = tokenizer(text, return_tensors="pt", max_length=512)

output = model(**sample_input)

print(torch.argmax(output[0], dim=-1))
```

Is there anything I am missing, I tried with some txt files from the dataset as well however I faced a max_seq_length issue and got only LABEL_8 and LABEL_10 with truncation
What is the exact mapping of LABEL_* mentioned in config.json of these models, Which is title, table, list etc.
![closing-disclosure-H25B-1 aec3e9325e5b](https://user-images.githubusercontent.com/57478479/93349017-980ab000-f854-11ea-8dae-fac93d62dbe5.png)

**Comment by liminghao1630** [comment]
Please refer to the file "labels.txt" I just uploaded.

**Comment by nishchay47b** [comment]
@liminghao1630 Thanks for the response. Am I missing anything in the above code? I am just trying to use BERT/Roberta models for inferencing but I am always getting LABEL_8. And any suggestion on how to get around max_seq_length, because generally, documents are full-page text. 
This might sound very basic, but I am just getting started so any suggestion will be very useful. Thanks.

**Comment by liminghao1630** [comment]
The LABEL_8 corresponds to the "Paragraph" label, which should be the most common one in an article. For the max_seq_length, my advice is to split all the tokens of a page into several sequences to avoid the length exceeding it.

**Comment by nishchay47b** [comment]
Thanks!
